Source/Data/BPMFMappings.txt

```diff
@@ -80901,7 +80901,7 @@
 漆樹 ㄑㄧ ㄕㄨˋ
 漆汁 ㄑㄧ ㄓ
 漆畫 ㄑㄧ ㄏㄨㄚˋ
-漆黑 ㄑㄩˋ ㄏㄟ
```

Comment:
suggestion: may we keep both? I believe some dictionaries have `ㄑㄩˋ ㄏㄟ`. 

https://www.moedict.tw/%E6%BC%86%E9%BB%91
https://dict.revised.moe.edu.tw/dictView.jsp?ID=103268&q=1&word=%E6%BC%86%E9%BB%91

Reply:
Fixed

Reply:
:heavy_check_mark: